scaladoc/test/dotty/tools/scaladoc/signatures/TranslatableSignaturesTestCases.scala

```diff
@@ -39,7 +39,7 @@ class PackageObjectSymbolSignatures extends SignatureTest("packageObjectSymbolSi
 class MergedPackageSignatures extends SignatureTest("mergedPackage", SignatureTest.all.filterNot(_ == "object"),
   sourceFiles = List("mergedPackage1", "mergedPackage2", "mergedPackage3"))
 
-class ExtensionMethodSignature extends SignatureTest("extensionMethodSignatures", SignatureTest.all)
+class ExtensionMethodSignature extends SignatureTest("extensionMethodSignatures", SignatureTest.all.filterNot(_ == "extension"))
```

Comment:
How come we're filtering out the `extension` ones now?

Reply:
As an extension has no name, I have no way to differentiate them for the documentation system when they have the same signature. I have only found this way, or change the signatures to be different.